standard_template/standard/clause_specification_text.adoc

```diff
@@ -574,12 +575,12 @@ Example of a domain object with [`"Trajectory"`][domain-types] domain type:
 
 ### 6.2. NdArray Objects
 
-A CoverageJSON object with the type `"NdArray"` is an NdArray object. It represents a multidimensional (>= 0D) array with named axes, encoded as a flat one-dimensional array in row-major order.
+A CoverageJSON object with the type `"NdArray"` is an NdArray object. It represents a multidimensional (>= 1D) array with named axes, encoded as a flat one-dimensional array in row-major order.
 
 - An NdArray object MUST have a member with the name `"values"` where the value is an array of numbers and nulls, or strings and nulls, where nulls represent missing data.
```

Comment:
```suggestion
- An NdArray object MUST have a member with the name `"values"` where the value is a non-empty array of numbers and nulls, or strings and nulls, where nulls represent missing data.
```

While the domain axes wouldn't allow empty ranges it makes sense to forbid empty arrays completely at this layer.

Reply:
Agreed and done in latest commit